tests/test_charset.py

```diff
@@ -104,299 +125,323 @@ def test_invalid_character_set(self, allow_reading_invalid_values):
-            pydicom.charset.decode_element(elem, ['ISO IR 192'])
-            assert 'Buc^Jérôme' == elem.value
-
-        elem = DataElement(0x00100010, 'PN', b'Buc^J\xe9r\xf4me')
-        with pytest.warns(UserWarning,
-                          match='Incorrect value for Specific Character Set '
-                                "'ISO-IR 144' - assuming 'ISO_IR 144'") as w:
-            pydicom.charset.decode_element(elem, ['ISO_IR 100', 'ISO-IR 144'])
+        elem = DataElement(0x00100010, "PN", b"Buc^J\xc3\xa9r\xc3\xb4me")
+        with pytest.warns(
+            UserWarning,
+            match="Incorrect value for Specific Character Set "
+            "'ISO IR 192' - assuming 'ISO_IR 192'",
+        ):
+            pydicom.charset.decode_element(elem, ["ISO IR 192"])
+            assert "Buc^Jérôme" == elem.value
+
+        elem = DataElement(0x00100010, "PN", b"Buc^J\xe9r\xf4me")
+        with pytest.warns(
+            UserWarning,
+            match="Incorrect value for Specific Character Set "
+            "'ISO-IR 144' - assuming 'ISO_IR 144'",
+        ) as w:
+            pydicom.charset.decode_element(elem, ["ISO_IR 100", "ISO-IR 144"])
             # make sure no warning is issued for the correct value
             assert 1 == len(w)
 
         # not patched incorrect encoding is replaced by default encoding
-        elem = DataElement(0x00100010, 'PN', b'Buc^J\xc3\xa9r\xc3\xb4me')
-        with pytest.warns(UserWarning,
-                          match="Unknown encoding 'ISOIR 192' - "
-                                "using default encoding instead"):
-            pydicom.charset.decode_element(elem, ['ISOIR 192'])
+        elem = DataElement(0x00100010, "PN", b"Buc^J\xc3\xa9r\xc3\xb4me")
+        with pytest.warns(
+            UserWarning,
+            match="Unknown encoding 'ISOIR 192' - " "using default encoding instead",
+        ):
+            pydicom.charset.decode_element(elem, ["ISOIR 192"])
 
         # Python encoding also can be used directly
-        elem = DataElement(0x00100010, 'PN', b'Buc^J\xc3\xa9r\xc3\xb4me')
-        pydicom.charset.decode_element(elem, ['utf8'])
-        assert 'Buc^Jérôme' == elem.value
+        elem = DataElement(0x00100010, "PN", b"Buc^J\xc3\xa9r\xc3\xb4me")
+        pydicom.charset.decode_element(elem, ["utf8"])
+        assert "Buc^Jérôme" == elem.value
 
     def test_patched_code_extension_charset(self):
         """Test some commonly misspelled charset values for code extensions."""
-        elem = DataElement(0x00100010, 'PN',
-                           b'Dionysios=\x1b\x2d\x46'
-                           b'\xc4\xe9\xef\xed\xf5\xf3\xe9\xef\xf2')
-        # correct encoding
-        pydicom.charset.decode_element(
-            elem, ['ISO 2022 IR 100', 'ISO 2022 IR 126']
+        elem = DataElement(
+            0x00100010,
+            "PN",
+            b"Dionysios=\x1b\x2d\x46" b"\xc4\xe9\xef\xed\xf5\xf3\xe9\xef\xf2",
```

Comment:
Black collapsed formerly multi-line b"..." together to one line, but didn't remove the extra b"".  I don't mind it though, especially with some of the following ones it kind of groups them in readable chunks.